kubernetes/src/test/java/oracle/kubernetes/operator/helm/HelmOperatorValues.java

```diff
@@ -100,6 +119,17 @@ private void addDomainsNamespaces(HashMap<String, Object> map) {
     }
   }
 
+  private void addOperatorImagePullSecrets(HashMap<String, Object> map) {
+    String secretName = getWeblogicOperatorImagePullSecretName();
+    if (secretName != null && (!secretName.isEmpty())) {
```

Comment:
Prefer a function from Guava: 

Strings.isNullOrEmpty(secretName)

returns true for both null and ""